mmaction/models/recognizers/recognizer2d.py

```diff
@@ -15,6 +17,14 @@ def forward_train(self, imgs, labels, **kwargs):
         losses = dict()
 
         x = self.extract_feat(imgs)
+
+        if self.backbone_from == 'torchvision':
+            if len(x.shape) == 4 and (x.shape[2] > 1 or x.shape[3] > 1):
+                # apply adaptive avg pooling
+                x = nn.AdaptiveAvgPool2d(1)(x)
```

Comment:
Why this is needed

Reply:
The shapes of different torchvision models' outputs vary, it may be N x C x H x W, N x C x 1 x 1 or N x C.

Reply:
I'm not sure if the special-cased processing is expected. Why backbones from other sources  are not handled?

Reply:
Backbones from other sources are written by us (mmcls / mmaction2), thus are more standard. In torchvision, there can be some  legacy issues.

Reply:
Okay